packages/fastify-renderer/src/node/utils.ts

```diff
@@ -21,10 +21,10 @@ const longestCommonPrefix = (...input: string[]) => {
 
 const cwd = process.cwd()
 
-export const mapFilepathToEntrypointName = (filepath: string) => {
+export const mapFilepathToEntrypointName = (filepath: string, base = '') => {
   const prefix = longestCommonPrefix(cwd, filepath)
   filepath = filepath.slice(prefix.length)
-  return filepath.replace(/\//g, '~')
+  return base.replace(/\//g, '~') + filepath.replace(/\//g, '~')
```

Comment:
This makes sense, is there a way to write a test for this tho? Or maybe just exercise the build command in CI?

Reply:
I don't see any straightforward way to write a test for this, building in CI is a good enough alternative though